lib/Github/Api/Apps.php

```diff
@@ -9,6 +9,13 @@
  */
 class Apps extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    final public function configurePreviewHeader()
```

Comment:
Just mark it private (and remove the final), there is no need to expose this method to the public api.

Reply:
```suggestion
    private function configurePreviewHeader()
```